source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.Reference/nuget/build/netstandard2.0/SkiaSharp.Views.Uno.targets

```
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+
+  <ItemGroup>
+    <UnoRuntimeEnabledPackage Include="SkiaSharp.Views.Uno" PackageBasePath="$(MSBuildThisFileDirectory)" />
```

@jeromelaban How does the `PackageBasePath` work. If this is included into the nuget package under the `build/netstandard` folder, the value will be that folder. Is this correct?

---

It's [used here](https://github.com/unoplatform/uno/blob/d08c0d6891331edb97e0d40bb4d11a0a2a9734eb/build/uno.winui.targets#L84), I still have to document it. That's correct, there's a missing `../`/
```suggestion
    <UnoRuntimeEnabledPackage Include="SkiaSharp.Views.Uno" PackageBasePath="$(MSBuildThisFileDirectory)../" />
```

I could not find a way to determine the location of the msbuild file declaring the item, so this is pretty much required for now :)